fabfile/deploy.py

```diff
@@ -1,13 +1,30 @@
+import os
+
 from fabric.context_managers import settings
 from fabric.decorators import task
 from fabric.operations import local
-from context_managers import ansible, in_project
 
 __author__ = 'deanmercado'
 
+
 @task
-def deploy(environment):
-    with ansible():
-        with in_project():
-            with settings(warn_only=True):
-                local('ansible-playbook -i orchestration/inventory/{} orchestration/site.yml'.format(environment))
+def deploy():
+    """
+    Starts deploy process
+    :return:
+    """
+    ENVIRONMENT_OPTIONS = {}
+    print "Option\tInventory"
```

Comment:
let's use `print()` throughout yurt - probably will want to switch it to python 3 when our dependencies support it